easybuild/easyblocks/f/fdtd_solutions.py

```diff
@@ -40,7 +44,23 @@ def configure_step(self):
 
     def build_step(self):
         """No build step for FDTD Solutions."""
-        pass
+        # locate RPM and rebuild it to make it relocatable
+        rpms = glob.glob(os.path.join(self.cfg['start_dir'], 'rpm_install_files', 'FDTD-%s*.rpm' % self.version))
```

Comment:
any particular reason why you need `glob` here? There is a 'random' part in the filename?